addons/sale/models/sale_order.py

```diff
@@ -1672,10 +1672,12 @@ def _notify_get_recipients_groups(self, message, model_description, msg_vals=Fal
         return groups
 
     def _notify_by_email_prepare_rendering_context(self, message, msg_vals=False, model_description=False,
-                                                   force_email_company=False, force_email_lang=False):
+                                                   force_email_company=False, force_email_lang=False,
+                                                   force_record_name=False):
         render_context = super()._notify_by_email_prepare_rendering_context(
             message, msg_vals=msg_vals, model_description=model_description,
-            force_email_company=force_email_company, force_email_lang=force_email_lang
+            force_email_company=force_email_company, force_email_lang=force_email_lang,
+            force_record_name=force_record_name,
         )
```

Comment:
Wouldn't it be finally time to migrate to `*args, **kwargs` so that we don't have to update all overrides every time you change something in the base method? :eyes: